docs/build-ansible.rst

```diff
@@ -198,7 +203,7 @@ Announcing Ansible
 * Add any info specific to this release.
 
   * Send any important information (like one-off changes to the release schedule) from here to
-  
+
```

Comment:
I guess that extra line is unintended (might break the bullet point formatting)

Reply:
It should not have any effect on formatting.

Reply:
yeah, I think my editor wiped out the two spaces on save. If they're not necessary I'd probably leave them out, but I don't have much of an opinion either way.